snapcraft/commands/registries.py

```diff
@@ -69,3 +69,41 @@ def run(self, parsed_args: "argparse.Namespace"):
             name=parsed_args.name,
             output_format=parsed_args.format,
         )
+
+
+class StoreEditRegistriesCommand(craft_application.commands.AppCommand):
+    """Edit a registries set."""
+
+    name = "edit-registries"
+    help_msg = "Edit registries"
```

Comment:
Maybe something that's not the same as the option? "Edit a registries set" could work, or even better if you could change the word "edit" to something different.